cluster-autoscaler/cloudprovider/openshift-machine-api/controller.go

```diff
@@ -14,19 +14,19 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package clusterapi
+package openshift_machine_api
```

Comment:
We're not running golint on this, but this would fail because of the underscores. I don't know if we care in this instance, but worth mentioning.

Reply:
This has gone away now.